packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/Alias/Alias.tsx

```diff
@@ -13,7 +13,7 @@ type IAliasProps = {
   editable: boolean;
 };
 
-export const Alias = observer(({ editable }: IAliasProps) => {
+export const Alias = observer<IAliasProps>(({ editable }) => {
```

Comment:
This tells the `observer` function which Props can be expected and will give a better IDE completion when now using trying to implement the component.

Reply:
What's the concrete better IDE completion? I don't really see a difference in my autocompletes.

Reply:
When you’re using `Alias` it knows the propstoo instead of only knowing it’s a `ComponentType`.
So basically the type is `ComponentType<Props>` instead of `ComponentType`.